salt/utils/install_pip.sls

```diff
@@ -1,6 +1,6 @@
 {% set python_dependencies = salt.grains.filter_by({
     'default': {
-      'python_libs': ['testinfra']
+      'python_libs': ['testinfra', 'python-pyinotify']
```

Comment:
This should be the name of the package as it is represented in PyPI. The `python-pyinotify` package is how it is represented in the Debian apt repository. For PyPI it's `pyinotify`